content/questions/reduce-array/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Reduce Array
```

Comment:
Can we change this to reduce-object? Perhaps rename the file too?

Reply:
Now renamed.